dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A PKCS#11 wrapper library"
+DESCRIPTION="libp11 is a library implementing a thin layer on top of PKCS#11 \
+API to make using PKCS11 implementations easier"
+HOMEPAGE="https://github.com/opensc/libp11"
+COPYRIGHT="2016 Michal Trojnara
+		2015-2016 Nikos Mavrogiannopoulos
+		2007-2009 Andreas Jellinghaus
+		2011 Martin Paljak"
```

Comment:
Please reduce the indentation of lines 6 to 8 by one tab.

BTW, the L in "Michał" is not the "usual" lowercase L, it is a variant with an sort of very small slash over it.
Since HaikuPorts accepts unicode chars in recipes, it would be nice to use the correct name.
You can copy the correct lowercase L from this comment, as well as from my [this message](#discussion_r97002736), or from the NEWS file
Sorry for being picky about this small detail. Thanks!